src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Dynamo.ViewModels;
+using Dynamo.Wpf.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace Dynamo.UI.Controls
+{
+    /// <summary>
+    /// Interaction logic for IncanvasLibrarySearchControl.xaml
+    /// </summary>
+    public partial class IncanvasSearchControl : UserControl
+    {
+        public IncanvasSearchControl()
+        {
+            InitializeComponent();
+        }
+
+        private SearchViewModel viewModel
```

Comment:
A property should be named with Pascal case: `SearchViewModel` rather than `viewModel`.